---
title: "day 2 of NTRES-6100"


date: "August 29, 2024"
layout: post
---

<script src="{{ site.url }}{{ site.baseurl }}/knitr_files/day-2-ntres_files/header-attrs-2.29/header-attrs.js"></script>

<section class="main-content">
<div id="recap" class="section level1">
<h1>recap</h1>
<p>day 2 of class was a success! we talked about markdown and how to
create a file. because I have been noodling with this website for a bit,
i decided to do some things differently from the file we made in class
so i could upload it here.<br />
<br> I hope you’ll enjoy this small insight into my mind hahaha <br></p>
</div>
<div id="oiiiiiiiii" class="section level1">
<h1>oiiiiiiiii</h1>
</div>
<div id="my-favorite-websites" class="section level1">
<h1>my favorite websites</h1>
<ol style="list-style-type: decimal">
<li><a href="http://inaturalist.org">iNaturalist</a></li>
<li><a href="http://discogs.com">discogs</a></li>
<li><a href="https://www.facebook.com/marketplace/?ref=app_tab">facebook
marketplace</a> <br> <br></li>
</ol>
<p>check out my indie collection</p>
<p><img src="https://i.redd.it/avtnnqio03ld1.jpeg" alt="I might have an addiction" width="600"/></p>
<p><br> <br></p>
<div id="yall-rock-with-tater-tot-hotdish" class="section level2">
<h2>y’all rock with tater tot hotdish???</h2>
<p><br> <br></p>
<p><img src="{{ site.url }}{{ site.baseurl }}/images/hotdish.jpeg"
alt="look at this shiiiiii yoooooo" /> <br> <br></p>
</div>
<div
id="this-is-a-script-to-prune-vcf-files-to-a-certain-number-of-snps."
class="section level2">
<h2>this is a script to prune VCF files to a certain number of SNPs.
<br></h2>
<p>I was using this to reduce my combined VCF files to make my genome
datasets more manageable at later steps</p>
<div class="sourceCode" id="cb1"><pre
class="sourceCode bash"><code class="sourceCode bash"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a><span class="ex">/programs/plink-1.9-x86_64-beta7/plink</span> <span class="at">--vcf</span> Primula_353super_LD_pruned_SNPs.recode.vcf <span class="at">--make-bed</span> <span class="at">--allow-extra-chr</span> <span class="at">--double-id</span> <span class="at">--out</span> Primula_genomeLD_pruned_thin</span>
<span id="cb1-2"><a href="#cb1-2" tabindex="-1"></a></span>
<span id="cb1-3"><a href="#cb1-3" tabindex="-1"></a><span class="ex">/programs/plink-1.9-x86_64-beta7/plink</span> <span class="at">--bfile</span> Primula_genomeLD_pruned_thin <span class="at">--thin-count</span> 30000 <span class="at">--allow-extra-chr</span> <span class="at">--double-id</span> <span class="at">--recode</span> <span class="at">--out</span> Primula_genomeLD_pruned_thin_plink</span>
<span id="cb1-4"><a href="#cb1-4" tabindex="-1"></a></span>
<span id="cb1-5"><a href="#cb1-5" tabindex="-1"></a><span class="ex">/programs/plink-1.9-x86_64-beta7/plink</span> <span class="at">--file</span> Primula_genomeLD_pruned_thin_plink <span class="at">--allow-extra-chr</span> <span class="at">--recode</span> vcf <span class="at">--out</span> Primula_genomeLD_pruned_thin_thinned</span>
<span id="cb1-6"><a href="#cb1-6" tabindex="-1"></a></span>
<span id="cb1-7"><a href="#cb1-7" tabindex="-1"></a></span>
<span id="cb1-8"><a href="#cb1-8" tabindex="-1"></a><span class="co">#gives missing proportion of loci for each individual</span></span>
<span id="cb1-9"><a href="#cb1-9" tabindex="-1"></a><span class="ex">/local/workdir/jbl256/Installed_programs/vcftools-0.1.16/bin/vcftools</span> <span class="at">--vcf</span> Primula_genomeLD_pruned_thin_thinned.vcf <span class="at">--missing-indv</span></span>
<span id="cb1-10"><a href="#cb1-10" tabindex="-1"></a></span>
<span id="cb1-11"><a href="#cb1-11" tabindex="-1"></a><span class="co">#average depth for each individual</span></span>
<span id="cb1-12"><a href="#cb1-12" tabindex="-1"></a><span class="ex">/local/workdir/jbl256/Installed_programs/vcftools-0.1.16/bin/vcftools</span> <span class="at">--vcf</span> Primula_genomeLD_pruned_thin_thinned.vcf <span class="at">--depth</span> </span>
<span id="cb1-13"><a href="#cb1-13" tabindex="-1"></a></span>
<span id="cb1-14"><a href="#cb1-14" tabindex="-1"></a><span class="co">#observed and expected heterozygosity</span></span>
<span id="cb1-15"><a href="#cb1-15" tabindex="-1"></a><span class="ex">/local/workdir/jbl256/Installed_programs/vcftools-0.1.16/bin/vcftools</span> <span class="at">--vcf</span> Primula_genomeLD_pruned_thin_thinned.vcf <span class="at">--het</span></span>
<span id="cb1-16"><a href="#cb1-16" tabindex="-1"></a></span>
<span id="cb1-17"><a href="#cb1-17" tabindex="-1"></a></span>
<span id="cb1-18"><a href="#cb1-18" tabindex="-1"></a><span class="co">#create fasta file for downstream analyses</span></span>
<span id="cb1-19"><a href="#cb1-19" tabindex="-1"></a><span class="ex">/local/workdir/jbl256/Installed_programs/vcf2phylip-2.8/vcf2phylip.py</span> <span class="at">--input</span> Primula_genomeLD_pruned_thin_thinned.vcf <span class="at">-f</span> <span class="at">-n</span></span>
<span id="cb1-20"><a href="#cb1-20" tabindex="-1"></a></span>
<span id="cb1-21"><a href="#cb1-21" tabindex="-1"></a><span class="co">#make structure file for later</span></span>
<span id="cb1-22"><a href="#cb1-22" tabindex="-1"></a><span class="ex">/local/workdir/jbl256/Installed_programs/stacks-2.66/bin/populations</span> <span class="at">-V</span> Primula_genomeLD_pruned_thin_thinned.vcf <span class="at">-O</span> ./ <span class="at">-M</span> popmap.txt <span class="at">--threads</span> 4 <span class="at">--ordered-export</span> <span class="at">--structure</span></span></code></pre></div>
</div>
</div>
</section>
