---
title: day 2 of NTRES-6100
date: August 29, 2024
output: 
  prettydoc::html_pretty:
    theme: cayman
    highlight: github
---
# recap
day 2 of class was a success! we talked about markdown and how to create a file. because I have been noodling with this website for a bit, i decided to do some things differently from the file we made in class so i could upload it here.  
<br>
I hope you'll enjoy this small insight into my mind hahaha
<br>

# oiiiiiiiii

# my favorite websites
1. [iNaturalist](http://inaturalist.org)
2. [discogs](http://discogs.com)
3. [facebook marketplace](https://www.facebook.com/marketplace/?ref=app_tab)
<br>
<br>

check out my indie collection

<img src="https://i.redd.it/avtnnqio03ld1.jpeg" alt="I might have an addiction" width="600"/>

<br>
<br>

## y'all rock with tater tot hotdish???

<br>
<br>

<img src="/images/hotdish.jpeg" alt="look at this shiiiiii yoooooo" width="600"/>

<br>
<br>

## this is a script to prune VCF files to a certain number of SNPs. <br>
I was using this to reduce my combined VCF files to make my genome datasets more manageable at later steps

```{bash, eval = FALSE}
/programs/plink-1.9-x86_64-beta7/plink --vcf Primula_353super_LD_pruned_SNPs.recode.vcf --make-bed --allow-extra-chr --double-id --out Primula_genomeLD_pruned_thin

/programs/plink-1.9-x86_64-beta7/plink --bfile Primula_genomeLD_pruned_thin --thin-count 30000 --allow-extra-chr --double-id --recode --out Primula_genomeLD_pruned_thin_plink

/programs/plink-1.9-x86_64-beta7/plink --file Primula_genomeLD_pruned_thin_plink --allow-extra-chr --recode vcf --out Primula_genomeLD_pruned_thin_thinned


#gives missing proportion of loci for each individual
/local/workdir/jbl256/Installed_programs/vcftools-0.1.16/bin/vcftools --vcf Primula_genomeLD_pruned_thin_thinned.vcf --missing-indv

#average depth for each individual
/local/workdir/jbl256/Installed_programs/vcftools-0.1.16/bin/vcftools --vcf Primula_genomeLD_pruned_thin_thinned.vcf --depth 

#observed and expected heterozygosity
/local/workdir/jbl256/Installed_programs/vcftools-0.1.16/bin/vcftools --vcf Primula_genomeLD_pruned_thin_thinned.vcf --het


#create fasta file for downstream analyses
/local/workdir/jbl256/Installed_programs/vcf2phylip-2.8/vcf2phylip.py --input Primula_genomeLD_pruned_thin_thinned.vcf -f -n

#make structure file for later
/local/workdir/jbl256/Installed_programs/stacks-2.66/bin/populations -V Primula_genomeLD_pruned_thin_thinned.vcf -O ./ -M popmap.txt --threads 4 --ordered-export --structure

```